admin/worker/run_autoscaler.go

```diff
@@ -61,7 +62,7 @@ func (w *Worker) runAutoscaler(ctx context.Context) error {
 			break
 		}
 
-		w.logger.Debug("succeeded in autoscaling:", zap.String("project_id", updatedProject.Name), zap.Int("project_slots", updatedProject.ProdSlots))
+		w.logger.Info("succeeded in autoscaling:", zap.String("project_id", updatedProject.Name), zap.Int("project_slots", updatedProject.ProdSlots))
```

Comment:
I think it better to leave a log line production env when slots for a project is updated.